model/gen-ai/spans.yaml

```diff
@@ -122,3 +122,65 @@ groups:
       - ref: server.port
         requirement_level:
           conditionally_required: If not default (443).
+
+  - id: span.gen_ai.client.create_agent
+    type: span
+    stability: experimental
+    brief: >
+      Describes GenAI agent creation and is usually applicable when working
+      with agent services.
+    note: |
+      The `gen_ai.operation.name` MUST be `create_agent`.
```

Comment:
do you feel strong about MUST? It bites us every time we're using it, so I'm suggesting to change it to SHOULD.

I don't have a specific scenario in mind where we would need something else, but given that we have 10+ different things that use different terminology and evolve very fast, I doubt that being strict here is possible, especially considering that we can't strictly define what "agent" is and what "creation" means

Reply:
Done, updated to `SHOULD`